editor/src/components/navigator/dependency-list.tsx

```diff
@@ -470,8 +470,8 @@ const AddTailwindButton = (props: AddTailwindButtonProps) => {
   }, [dispatch])
 
   const tailwindAlreadyAdded =
-    props.packagesWithStatus.find((p) => p.name === 'tailwindcss') &&
-    props.packagesWithStatus.find((p) => p.name === 'postcss')
+    props.packagesWithStatus.find((p) => p.name === 'tailwindcss') != null &&
```

Comment:
```suggestion
    props.packagesWithStatus.some((p) => p.name === 'tailwindcss') &&
```

Reply:
I'm assuming it's an array here I have to admit.

Reply:
Ha! Yes, that makes much more sense

Reply:
Fixed